swarm/api/routes/db.py

```
@@ -283,6 +295,11 @@ async def ingest_run_events(run_id: str):
     Returns:
         Dict with ingestion statistics.
     """
+    try:
+        run_id = validate_path_component(run_id)
+    except ValueError as e:
+        raise HTTPException(status_code=400, detail=str(e))
```

Ruff (0.14.14)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)